lib/internal/url.js

```diff
@@ -286,11 +286,9 @@ class URLSearchParamsIterator {
     );
     const breakLn = StringPrototypeIncludes(inspect(output, innerOpts), '\n');
     const outputStrs = ArrayPrototypeMap(output, (p) => inspect(p, innerOpts));
-    let outputStr;
+    let outputStr = ` ${ArrayPrototypeJoin(outputStrs, ', ')}`;
```

Comment:
This impacts performance

Reply:
Do we have benchmarks for this code?

Reply:
I doubt it, but on the following if statement where it overwrites the value, one additional array operation is done. i'm 100% it impacts performance without any noticeable improvement.